Added file CreateShortcut for file's create shortcut use-case.

Manvendra-P-Singh opened the pull request:
# Description

Added file CreateShortcut for file's create shortcut use-case.It includes Google ADC as well.

Fixes # (211027701)

## Is it been tested?
- [X] Development testing done

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have performed a peer-reviewed with team member(s)
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

Manvendra-P-Singh committed:
Added file CreateShortcut for file's create shortcut use-case.

RajeshGogo reviewed (commented):
Added only comment and no function description or parameter details
Please add

Manvendra-P-Singh committed:
Added file CreateShortcut for file's create shortcut use-case.

Added file CreateShortcut for file's create shortcut use-case.It includes Google ADC as well.
